control/control_validator/src/utils.cpp

```diff
@@ -44,7 +44,7 @@ void insertPointInPredictedTrajectory(
 
 TrajectoryPoints reverseTrajectoryPoints(const TrajectoryPoints & trajectory_points)
 {
-  TrajectoryPoints reversed_trajectory_points(trajectory_points.size());
+  TrajectoryPoints reversed_trajectory_points(0);
```

Comment:
@kei-t4 
thank you for detecting my stupid bug.
I think what I wanted to do is here.
```suggestion
  TrajectoryPoints reversed_trajectory_points;
  reversed_trajectory_points.reserve(trajectory_points.size());
```

And setting the vector size to 0 in the constructor is unnecessary, as the default constructor already creates an empty vector.

Reply:
@kyoichi-sugahara 

You are mentioning to me in this message, but I think it should be not me, but should someone you work with.
Please check the message recipient. 
Thanks.